pwnlib/libcdb.py

```diff
@@ -86,16 +117,16 @@ def query_libc_rip(params):
         return None
 
 # https://libc.rip/
-def provider_libc_rip(hex_encoded_id, hash_type):
+def provider_libc_rip(search_target, search_type):
     # Build the request for the hash type
     # https://github.com/niklasb/libc-database/blob/master/searchengine/api.yml
-    if hash_type == 'build_id':
-        hash_type = 'buildid'
-    params = {hash_type: hex_encoded_id}
+    if search_type == 'build_id':
+        search_type = 'buildid'
+    params = {search_type: search_target}
```

Comment:
This actually uses `id` as identifier instead of `libs_id`. That's where the name came from. If we invent `libs_id` now, this has to be mapped here.